bin/git-fork

```diff
@@ -54,9 +87,30 @@ if [ "$origin" = true ]; then
     git fetch origin
 else
     # clone forked repo into current dir
-    git clone "${remote_prefix}${user}/${project}.git" "$project"
-    # add reference to origin fork so can merge in upstream changes
-    cd "$project"
-    git remote add upstream "${remote_prefix}${owner}/${project}.git"
-    git fetch upstream
+    git clone "${remote_prefix}${user}/${project}.git" "$project" 2>/dev/null
```

Comment:
I agree that the output of `git clone` is cumbersome, but still think we should not hide the error message. They are like low level log, which may be useful sometimes.